src/code/z_sub_s.c

```diff
@@ -100,7 +158,21 @@
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_sub_s/func_8013E5CC.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_sub_s/func_8013E640.s")
+Actor* SubS_GetActorByFunction(GlobalContext* globalCtx, Actor* actor, Actor* actorListStart, u8 actorCategory,
```

Comment:
I'm not completely convinced with the `ByFunction` suffix. `ByCallback` may be better, but I'm not sure either. 
Would be nice to see what everybody else thinks about this.

Reply:
What about something like `ByVerification` to go along with how the function argument is called `verifyActor`?

Reply:
This function's pretty weird... it only seems to be used twice in the codebase: once in EnTk, with a verifier that always returns false, and once in ObjHunsui,
```c
? func_80B9D288(s32 arg0, ? arg1, void *arg2, ? arg3) {
    ? phi_v1;

    phi_v1 = 0;
    if ((arg2->unk1C & 0xF) == 8) {
        phi_v1 = 1;
    }
    return phi_v1;
}
```
i.e. the bottm nybble of params is 8.

Reply:
Note that I also changed the other names as well to follow the same format. Let me know if they are still good enough.